librz/core/rop.c

```diff
@@ -737,11 +822,14 @@ static void rz_rop_gadget_print_standard_mode(const RzCore *core, const RzRopGad
 			continue;
 		}
 		if (reg_info->is_var_write) {
-			rz_cons_printf("Var write: %s Initial value: %llu New Value: %llu\n", reg_info->name, reg_info->init_val, reg_info->new_val);
+			rz_cons_printf("Var write: %s Initial value: 0x%llx New Value: 0x%llx\n", reg_info->name, reg_info->init_val, reg_info->new_val);
```

Comment:
You need to use `PFMT64x` etc instead of `llx`. Otherwise it is not portable to Windows.

Reply:
Also below